spec/support/coronavirus_landing_page_steps.rb

```diff
@@ -98,25 +98,14 @@ def then_i_can_see_the_timeline_for_wales
 
   def then_i_can_see_the_accordions
     expect(page).to have_selector("h2", text: "Guidance and support")
-    expect(page).to have_selector(".gem-c-accordion__section-header", text: "How to protect yourself and others")
   end
 
   def then_i_can_see_the_business_accordions
     expect(page).to have_selector("h2", text: "Guidance and support")
-    expect(page).to have_selector(".gem-c-accordion__section-header", text: "Funding and support")
```

Comment:
This method is not being used, so can be completely removed.

Reply:
Ah of course because the hub pages don't exist anymore. Thanks, good spot, have updated.